### ISSUE TITLE
gfs cluster does not recover

### ISSUE THREAD

**Issue by rimusz** [neshte/gluster-k8s-sidecar#1]
hey there, nice work, I was looking to make a similar solution by myself too.
tried to bootstrap 2 nodes gsf cluster, worked fine, so for the test deleted one gfs pod, gfs cluster
did not come back :(

**Comment by rimusz** [comment]
got 4 peers gfs, cluster, tried to delete the third peer pod, cluster recovered, but deleting the first one, that never connected back to the cluster

**Comment by neshte** [comment]
yes, it has to be multiples of your replication factor, and yes recovery is not very stable... if you have pull requests are welcome!

**Comment by rimusz** [comment]
need to learn glusterfs first before suggesting any improvements to recovery part :)
